SQF/dayz_code/stringtable.xml

```diff
@@ -7475,7 +7475,7 @@
 		</Key>
 		<Key ID="str_death_shothead">
 			<English>a gunshot to the head.</English>
-			<German>durch einen Schuss in den Kopf.</German>
+			<German>durch einen Kopfschuss.</German>
```

Comment:
So the "Schuss in den Kopf"?
It's free to discuss, it was the suggestion from Basti.